test/boost/cql_query_test.cc

```diff
@@ -91,6 +91,38 @@ SEASTAR_TEST_CASE(test_create_table_with_id_statement) {
     });
 }
 
+SEASTAR_TEST_CASE(test_create_twcs_table_no_ttl) {
+    return do_with_cql_env([](cql_test_env& e) {
+        return seastar::async([&e] {
```

Comment:
nitpick: there's also `do_with_cql_env_thread`, which lets you avoid explicitly calling a thread